src/m365/spo/commands/homesite/homesite-set.ts

```diff
@@ -22,41 +23,56 @@ class SpoHomeSiteSetCommand extends SpoCommand {
   }
 
   public get description(): string {
-    return 'Sets the specified site as the Home Site';
+    return 'Sets the specified site as the Home Site and optionally set the Viva Connections landing experience to the SharePoint Home Site';
   }
 
   constructor() {
     super();
-  
+
     this.#initOptions();
     this.#initValidators();
+    this.#initTypes();
   }
-  
+
   #initOptions(): void {
     this.options.unshift(
       {
         option: '-u, --siteUrl <siteUrl>'
+      },
+      {
+        option: '--vivaConnectionsDefaultStart [vivaConnectionsDefaultStart]'
       }
     );
   }
-  
+
   #initValidators(): void {
     this.validators.push(
       async (args: CommandArgs) => validation.isValidSharePointUrl(args.options.siteUrl)
     );
   }
 
+  #initTypes(): void {
+    this.types.boolean.push('vivaConnectionsDefaultStart');
+  }
+
   public async commandAction(logger: Logger, args: CommandArgs): Promise<void> {
     try {
       const spoAdminUrl = await spo.getSpoAdminUrl(logger, this.debug);
       const reqDigest = await spo.getRequestDigest(spoAdminUrl);
+      let requestData: string = `<Request AddExpandoFieldTypeSuffix="true" SchemaVersion="15.0.0.0" LibraryVersion="16.0.0.0" ApplicationName="${config.applicationName}" xmlns="http://schemas.microsoft.com/sharepoint/clientquery/2009">`;
```

Comment:
I'd opt we change the entire command to use the REST API. Are you willing to change it.

Reply:
@milanholemans : thanks for the detailed review and thank you for the great find using REST API to achieve the same outcome. Definitely the REST API looks simpler and will look into it refactoring the code.

Reply:
I have updated the other command to set home site to use REST API as well.